easybuild/easyblocks/generic/cmakepythonpackage.py

```diff
@@ -53,11 +53,6 @@ def extra_options(extra_vars=None):
         extra_vars['runtest'][0] = None
         return extra_vars
 
-    def __init__(self, *args, **kwargs):
-        """Initialize with PythonPackage."""
-        PythonPackage.__init__(self, *args, **kwargs)
-        self._lib_ext = None  # From CMakeMake.__init__
```

Comment:
Removing this makes sense to me, but maybe we should try and hunt down why we have it like this.
There may be a reason it's like this...

I'd say we definitely need to try and test this modified easyblock extensively (ideally with all easyconfigs we have that use `CMakePythonPackage`, directly or indirectly, although I can't come up with a reason why this would break something, from the top of my head...

Reply:
`git blame` shows that this code is very old. The addition with `_lib_ext` was done by me when I ran into this earlier and didn't wanted to touch anything else.   
So I believe it was rather missing experience (e.g. not knowing about `super()`) especially with multi-inheritance than intentional.

I did test with `GenomeWorks-2021.02.2-fosscuda-2020b.eb` (where I encountered it) and also verified that now **both** `__init__` functions are called. Might make sense to test some more but I guess for "plain" ECs using this block there won't be any issues if that one worked so rather other easyblocks deriving from this may be affected although I'd consider that a bug in those as I'm very sure this is correct (now).